src/Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs

```diff
@@ -632,7 +639,7 @@ private void FilteredWriteObject(PSObject obj, List<PSNoteProperty> addedNotePro
                 bool isObjUnique = true;
                 foreach (UniquePSObjectHelper uniqueObj in _uniques)
                 {
-                    ObjectCommandComparer comparer = new(true, CultureInfo.CurrentCulture, true);
+                    ObjectCommandComparer comparer = new(true, CultureInfo.CurrentCulture, !CaseInsensitive.IsPresent);
```

Comment:
Please use named argument for `true`.

Reply:
Added named arguments. Included for `ascending` and `caseSensitive` for better clarity.